jmix-localfs/localfs/src/main/java/io/jmix/localfs/LocalFileStorageProperties.java

```diff
@@ -37,4 +45,11 @@ public LocalFileStorageProperties(
     public String getStorageDir() {
         return storageDir;
     }
+
+    /**
+     * @see #disablePathCheck
+     */
+    public boolean isDisablePathCheck() {
```

Comment:
Why do we need this property?
Let's disable any attempt to request a file outside FileStorage root.